Header header
int32 id
float64 vel_x
float64 vel_y
geometry_msgs/Pose pose
geometry_msgs/Point32 dimension